src/index.js

```diff
@@ -1,207 +1,206 @@
-import { Component } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
 
-class DidomiSDK extends Component {
-  static propTypes = {
-    apiKey: PropTypes.string,
-    iabVersion: PropTypes.number,
-    noticeId: PropTypes.string,
-    config: PropTypes.object,
-    gdprAppliesGlobally: PropTypes.bool,
-    onReady: PropTypes.func,
-    onConsentChanged: PropTypes.func,
-    onNoticeShown: PropTypes.func,
-    onNoticeHidden: PropTypes.func,
-    onNoticeBackdropclick: PropTypes.func,
-    onNoticeClickAgree: PropTypes.func,
-    onNoticeClickMoreInfo: PropTypes.func,
-    onPreferencesClickAgreeToAll: PropTypes.func,
-    onPreferencesClickDisagreeToAll: PropTypes.func,
-    onPreferencesClickPurposeAgree: PropTypes.func,
-    onPreferencesClickPurposeDisagree: PropTypes.func,
-    onPreferencesClickViewVendors: PropTypes.func,
-    onPreferencesClickSaveChoices: PropTypes.func,
-    onPreferencesClickVendorAgree: PropTypes.func,
-    onPreferencesClickVendorDisagree: PropTypes.func,
-    onPreferencesClickVendorSaveChoices: PropTypes.func
-  }
-
-  static defaultProps = {
-    apiKey: null,
-    iabVersion: 1,
-    noticeId: null,
-    config: {},
-    gdprAppliesGlobally: true,
-    onReady: () => {},
-    onConsentChanged: () => {},
-    onNoticeShown: () => {},
-    onNoticeHidden: () => {},
-    onNoticeBackdropclick: () => {},
-    onNoticeClickAgree: () => {},
-    onNoticeClickMoreInfo: () => {},
-    onPreferencesClickAgreeToAll: () => {},
-    onPreferencesClickDisagreeToAll: () => {},
-    onPreferencesClickPurposeAgree: () => {},
-    onPreferencesClickPurposeDisagree: () => {},
-    onPreferencesClickViewVendors: () => {},
-    onPreferencesClickSaveChoices: () => {},
-    onPreferencesClickVendorAgree: () => {},
-    onPreferencesClickVendorDisagree: () => {},
-    onPreferencesClickVendorSaveChoices: () => {}
-  }
-
+const DidomiSDK = ({
+  apiKey: apiKeyProp =  null,
+  iabVersion =  1,
+  noticeId =  null,
+  config =  {},
+  gdprAppliesGlobally: gdprAppliesGloballyProp =  true,
+  onReady =  () => {},
+  onConsentChanged =  () => {},
+  onNoticeShown =  () => {},
+  onNoticeHidden =  () => {},
+  onNoticeBackdropclick =  () => {},
+  onNoticeClickAgree =  () => {},
+  onNoticeClickMoreInfo =  () => {},
+  onPreferencesClickAgreeToAll =  () => {},
+  onPreferencesClickDisagreeToAll =  () => {},
+  onPreferencesClickPurposeAgree =  () => {},
+  onPreferencesClickPurposeDisagree =  () => {},
+  onPreferencesClickViewVendors =  () => {},
+  onPreferencesClickSaveChoices =  () => {},
+  onPreferencesClickVendorAgree =  () => {},
+  onPreferencesClickVendorDisagree =  () => {},
+  onPreferencesClickVendorSaveChoices =  () => {},
+  src = "https://sdk.privacy-center.org/"
```

Comment:
We should also update the documentation with this new `prop`

Reply:
Done in new MR